src/util/DefaultValueProcessor.ts

```diff
@@ -22,13 +23,27 @@ export const addValues = (testCase: any): any => {
     throw new Error("Unable to parse Patient Resource");
   }
   const patientRef: Reference = { reference: `Patient/${patientId}` };
-
+  const conditionReferences: string[] = [];
+  resultJson?.entry?.forEach((entry) => {
```

Comment:
Having this loop here introduces unnecessary processing whenever the import doesn't have either a Condition or an Encounter. I'm thinking through alternatives.

Reply:
As discussed, we couldn't find a way that didn't require additional loops with the current implementation due to linking the encounter to the condition. Which is fine, importing is expected to be done sparingly.